### PR TITLE
Updated login test

### PR DESCRIPTION
**Pull request by rbillings**
I'm not sure why I merged the original login commit as half of it was missing. This fixes it so the test passes.

### CODE REVIEW

File: pages/base.py [hunk 1]
```diff
@@ -15,10 +19,27 @@ def header(self):
 
     class Header(PageRegion):
 
+        _login_locator = (By.CSS_SELECTOR, 'a.btn > span:nth-child(1)')
+        _logout_icon_locator = (By.ID, 'logoutLabel')
         _root_locator = (By.ID, 'th-global-navbar')
```
Comment (davehunt) [inline pages/base.py]: Nit: We should put the `_root_locator` at the top.

File: pages/base.py [hunk 2]
```diff
@@ -15,10 +19,27 @@ def header(self):
...
         _dropdown_menu_switch_page_locator = (By.CSS_SELECTOR, '.open ul > li a')
         _dropdown_menu_locator = (By.ID, 'th-logo')
 
+        @property
+        def is_user_logged_in(self):
+            return self.is_element_visible(self._logout_icon_locator)
+
+        def click_login(self):
+            self.selenium.find_element(*self._login_locator).click()
+            return not self.is_element_visible(self._login_locator)
```
Comment (davehunt) [inline pages/base.py]: This function doesn't need to return a value.

**Comment by davehunt** [comment]
This test doesn't appear to be running in Jenkins... perhaps it's being skipped due to the import error.

r+ with comments addressed.

**Comment by rbillings** [comment]
Commits look happy, merging.

**Pull request merged**